docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -88,8 +88,6 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 			// TODO: Also skip `@fluid-internal` packages once we no longer have public, user-facing APIs that reference their contents.
 			return ["@fluid-private"].includes(packageScope);
 		},
-		frontMatter: (apiItem) =>
```

Comment:
Note for reviewers: support for this in the documenter has been deprecated, so we will inject front-matter ourselves lower down.